internal/api/connectors/inventory/types.go

```diff
@@ -0,0 +1,13 @@
+package inventory
+
+type HostDetailsResponse struct {
```

Comment:
I would remove the Response from the struct name.  I like to think of interfaces/structs like this as a way to hide details about how the data is retrieved.  In other words, at this level, the rest of your code doesn't need to know about Requests and/or Responses.  By including Response in the name of the struct, you kind of let the detail leak out into other areas of your code.  "Leaky abstraction" is what this is called.